man/systemd.exec.xml

```diff
@@ -3520,6 +3520,19 @@ StandardInputData=V2XigLJyZSBubyBzdHJhbmdlcnMgdG8gbG92ZQpZb3Uga25vdyB0aGUgcnVsZX
         <literal>[]</literal> wildcards are not permitted, nor are <literal>*</literal> wildcards anywhere
         except at the end of the glob expression.</para>
 
+        <para>Optionally, the credential name or glob may be followed by a colon followed by a rename pattern.
+        If specified, all credentials matching the credential name or glob are renamed according to the given
+        pattern. For example, if <literal>ImportCredential=my.original.cred:my.renamed.cred</literal> is
+        specified, the service manager will read the <literal>my.original.cred</literal> credential and make
+        it available as the <literal>my.renamed.cred</literal> credential to the service. Similarly, if
+        <literal>ImportCredential=my.original.*:my.renamed.</literal> is specified, the service manager will
+        read all credentials starting with <literal>my.original.</literal> and make them available as
+        <literal>my.renamed.xxx</literal> to the service.</para>
+
+        <para>If <varname>ImportCredential=</varname> is specified multiple times with the same rename
+        pattern, credentials renamed by later patterns will override credentials renamed by earlier patterns
```

Comment:
With your change to hash ops it's no longer the case? Now it's possible to rename to multiple targets (I don't really see the use case though...)

Reply:
Isn't our `OrderedSet` ordered by insertion order? I have to admit I've never used it before.

Reply:
It is. But now `rename` is part of the hash too, so previous ImportCred= won't be overridden even with same glob.

Reply:
I don't see why we shouldn't support renaming to multiple targets, might as well support it I think.

Reply:
Please update the documentation then

Reply:
Ah I think I see where the confusion is. I mean override as in if two credentials happen to be the same after renaming the one matched by the later `ImportCredential=` will still get renamed over the earlier one. The `ImportCredential=` won't get overridden but the credentials will still end up overriding each other.